sdks/typescript/src/clients/rest/generated/Api.ts

```diff
@@ -1718,6 +1718,10 @@ export class Api<SecurityDataType = unknown> extends HttpClient<SecurityDataType
        * @maxLength 36
        */
       workflowId?: string;
+      /** The workflow name to get runs for. */
+      workflowName?: string;
+      /** The cron name to get runs for. */
```

Comment:
have you tested the sdk?

Reply:
ah turns out I forgot I did this - I forgot that the TS API wrapper in the crons client just passes the params along properly so no changes needed there. tested it and it works fine